praw/models/reddit/submission.py

```diff
@@ -396,6 +396,8 @@ class Submission(SubmissionListingMixin, UserContentMixin, FullnameMixin, Reddit
     Attribute                  Description
     ========================== =========================================================
     ``author``                 Provides an instance of :class:`.Redditor`.
+    ``author_flair_text``      The text content of the authors flair, or ``None`` if not
+                               flaired. 
```

Comment:
```suggestion
                               flaired.
```

Reply:
Seems that `black` really hates that stray space.

Reply:
You can run black locally so it doesn't fail in PR.

Reply:
Thanks.